src/components/Table/Table.scss

```diff
@@ -48,39 +36,37 @@ $block: '.#{variables.$ns}table';
         &_sortable {
             cursor: pointer;
             user-select: none;
-
-            &:hover {
-                #{$block}__sort-indicator {
-                    opacity: 1;
-                }
-            }
         }
 
         &:hover {
             #{$block}__resizer {
                 opacity: 1;
             }
         }
-
-        &_wide {
-        }
     }
 
-    &__group-title-wrapper {
+    &__group-title {
         position: sticky;
         inset-inline-start: 0;
-        padding: var(--g-spacing-1) 0;
+        margin: 0;
     }
 
-    &__group-title {
-        margin: 0;
+    &__group-title-arrow {
+        display: inline-block;
+        vertical-align: middle;
+        transform: rotate(-90deg);
+        transition: transform 0.1s ease-out;
+
+        &_down {
+            transform: rotate(0);
+        }
     }
 
     &__group-expand-button {
         appearance: none;
 
         display: flex;
-        gap: var(--g-spacing-2);
+        gap: 8px;
```

Comment:
Maybe we should create some css variables for spacing?

Reply:
There is no need. This component doesn't have any specific design. New styled table will use gravity/uikit styles and variables.